### DISCUSSION TITLE
8bitdo Neo Geo pad

### DISCUSSION THREAD

**Discussion by DragReduction** [darthcloud/BlueRetro#927] [Q&A] [answered]
Hi. 
Wondering if anyone can tell me if it’s possible to use the 8bitdo neo pad shoulder buttons as a turbo option?
I’ve tried the web config but no option to map the L R buttons.
Thank you.

**Comment by GrumpyGopher** [comment, marked as answer]
What version of the firmware are you running? I had a problem mapping the shoulder buttons in version 1.9.0 but I was able to map them in version 1.8.4.  I haven't tried the latest version

**Reply by DragReduction** [reply]
Hi. Thanks for the reply. I’m on the latest firmware. I’ll downgrade to the firmware you used and report back.

**Reply by DragReduction** [reply]
That worked perfectly.
Not working with latest firmware for some reason but happy to get it working now.
Thanks again.

**Reply by GrumpyGopher** [reply]
Awesome glad that work.  I had meant to post something when I found it but time got away from me.  I'll do some more tests and write something up.

**Reply by GrumpyGopher** [reply]
I created this issue to track the problem:
https://github.com/darthcloud/BlueRetro/issues/933

**Reply by DragReduction** [reply]
Thank you very much.

**Comment by darthcloud** [comment]
The next version will support the trigger properly.

You can try beta here:

https://github.com/darthcloud/BlueRetro/actions/runs/8680779369

**Reply by GrumpyGopher** [reply]
I had a chance to test the beta for HW1 using the universal firmware and all looks good mapping to L2 and R2

**Comment by Metro-City** [comment]
Sorry for the question, but which one do you use for blueretro neo geo cd?
hw1? BlueRetro_hw1_parallel_1p.bin and BlueRetro_hw1_parallel_2p.bin I'm not expert on these things, I bought this adapter and I hope someone will give me more information, thanks
